docs/reference/index.md

```diff
@@ -1,23 +1,26 @@
 # Schema reference
 
+The Infrastructure Project Level Data Specification is described using JSON Schema 0.4.
+
+The [schema file is available here](../../../../_static/project-level/project-schema.json) and can be explored using the [schema browser](schema.md).
+
+Full schema and codelist [reference documentation is also available](reference.md).
+
+The [packaging data section](project_package.md) describes how to package OC4IDS data for publication. 
+
+The OC4IDS schema reuses many of the building blocks from the OCDS schema, these are introduced in the [getting started section of the OCDS documentation](http://standard.open-contracting.org/latest/en/getting_started/).
```

Comment:
fixed in https://github.com/open-contracting/infrastructure/pull/89/commits/381b67bbdee4987956315a5b528d5b5f8538634e

I also used Open Contracting Data Standard rather than OCDS